run.py

```diff
@@ -239,6 +242,22 @@ def keyboard_click(*KEYBOARD_NAMES):
 def grab_screen(n):
     return callback.get_screen_grab()
 
+@app.callback(
+    Output('calendar-output-message', 'children'),
+    [Input('time-submit-button', 'n_clicks'),
+     Input('date-picker-range', 'start_date'),
+     Input('date-picker-range', 'end_date')],
```

Comment:
You can use it as State
```
State('date-picker-range', 'start_date'),
State('date-picker-range', 'end_date')
```

Otherwise this callback is triggered all the time and only clicks = 0 stop it from execute